server/src/grpc/server.rs

```diff
@@ -143,6 +150,12 @@ pub async fn grpc_server(config: Config) {
         .add_service(vertiport::RpcServiceServer::new(
             vertiport::GrpcServer::default(),
         ))
+        .add_service(itinerary::RpcServiceServer::new(
```

Comment:
These are already created earlier. I think they are a result from a merge.
You can remove row 153-158

Reply:
[cc969e4](https://github.com/Arrow-air/svc-storage/pull/62/commits/cc969e45bda8b46c79ae04c6c37063bdc700886e)